test/quantization/ao_migration/test_quantization_fx.py

```diff
@@ -11,8 +11,6 @@ def test_function_import_quantize_fx(self):
             '_check_is_graph_module',
             '_swap_ff_with_fxff',
             '_fuse_fx',
-            'Scope',
-            'ScopeContextManager',
```

Comment:
these should not be removed I think

Reply:
Done - I added Scope and ScopeContextManager back to the test and to torch/quantization/quantize_fx.py